Dockerfile

```diff
@@ -28,11 +28,33 @@ FROM base as development
 
 CMD [ "poetry", "run", "flask", "run", "--host=0.0.0.0", "--port=5000", "--debugger"] 
 
+# testing stage
+FROM base as test
+RUN pip install pytest
+RUN poetry add pytest --dev
+#WORKDIR /tests
+ENV GECKODRIVER_VER v0.31.0
+# Install the long-term support version of Firefox (and curl if you don't have it already)
+RUN apt-get update && apt-get install -y firefox-esr curl
+  
+# Download geckodriver and put it in the usr/bin folder
+RUN curl -sSLO https://github.com/mozilla/geckodriver/releases/download/${GECKODRIVER_VER}/geckodriver-${GECKODRIVER_VER}-linux64.tar.gz \
+   && tar zxf geckodriver-*.tar.gz \
+   && mv geckodriver /usr/bin/ \
+   && rm geckodriver-*.tar.gz
+RUN  pip install selenium   
+ENTRYPOINT ["poetry", "run", "pytest"]
+
 # docker build -f .\Dockerfile --tag todo-app .
 # docker run -p 8181:5000 --env-file .\.env todo-app
 
 # docker build --target development --tag todo-app:dev .
 # docker build --target production --tag todo-app:prod .
+# docker build --target test --tag todo-app:test .
 
 # docker run -d -p 8181:5000 --env-file .\.env --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:dev
 # docker run -d -p 8182:5000 --env-file .\.env --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:prod
+# docker run -d -p 8183:5000 --env-file .\.env --mount type=bind,source="$(pwd)"/todo_app,target=/app/todo_app todo-app:test
+# docker run --env-file .env.test todo-app:test tests
+# docker run todo-app:test tests_e2e
```

Comment:
I'm always inclined to suggest avoiding putting commands like these here; they tend to be quickly forgotten about and end up out of date, though I appreciate you caught that they needed updating here!

Reply:
i will leave for now but I appreciate the comment
Thanks